README.md

```diff
@@ -244,13 +244,13 @@ Please contact us if you have a new mining pool.
 - 2pool@tokyo: https://2pool.sugarchain.org
 - rplant@worldwide: https://pool.rplant.xyz
 - mofumofu@tokyo: https://nomp.mofumofu.me
-- dxpool@china: https://www.dxpool.com
-- anomp@china: http://39.103.144.156
+- dxpool@china: https://www.dxpool.net/?pool=labs
```

Comment:
it seems `https://www.dxpool.com/?pool=labs` still works

Reply:
dxpool.com is not available in some areas, so add dxpool.net instead